Dockerfile-debian.template

```diff
@@ -16,7 +16,13 @@
 # limitations under the License.
 ###############################################################################
 
-FROM openjdk:8-jre
+FROM openjdk:8-jre-slim
+
+# Install dependencies
+RUN set -ex; \
+  apt-get update; \
+  apt-get -y install gnupg libsnappy1v5 wget; \
```

Comment:
We can but we won't win a lot (apart of cleanliness) because we will write an en extra image layer just for the remove and it won't save any space with this.
Big motivation for this would be to separate the build requirements (gnupg/wget) from the runtime ones (libsnappy1v5), but again this will come at the cost of an extra layer for a small return.
Or am I missing something?